src/lib/browse/layers/PointAmenityLayerControl.svelte

```diff
@@ -38,11 +42,6 @@
     // TODO Outline?
   });
   let show = false;
-
-  function tooltip(feature: MapGeoJSONFeature): string {
```

Comment:
Although we didn't need it before, I definitely think it makes more sense for the specific layer control to determine how the tooltip is constructed